github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.31.yaml

```diff
@@ -0,0 +1,66 @@
+---
+presubmits:
+  k8snetworkplumbingwg/kubemacpool:
+    - name: pull-kubemacpool-unit-test-v0.31
+      branches:
+        - release-v0.31
+      annotations:
+        fork-per-release: "true"
+      always_run: true
+      optional: false
+      decorate: true
+      decoration_config:
+        timeout: 3h
+        grace_period: 5m
+      cluster: phx-prow
+      max_concurrency: 6
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
+          type: vm
+          zone: ci
+        containers:
+          - image: quay.io/kubevirtci/kubevirt-infra-bootstrap:v20201201-08dc4a9
+            securityContext:
+              privileged: true
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "automation/check-patch.unit-test.sh"
+
+    - name: pull-kubemacpool-e2e-k8s-v0.31
+      branches:
+        - release-v0.31
+      annotations:
+        fork-per-release: "true"
+      always_run: true
+      optional: false
+      decorate: true
+      cluster: prow-workloads
+      decoration_config:
+        timeout: 3h
+        grace_period: 5m
+      max_concurrency: 6
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
```

Comment:
The two jobs in this PR differ in the nodeSelector, its pretty weird 
https://github.com/kubevirt/project-infra/pull/1508/files#diff-7504aec980c9447b603f21a174e3acc4db1e4e5284ffa88134015defc4cd2e5fR22

Reply:
you are right. @phoracek this diff dates back to the very start of KMP lanes ( see [here](https://github.com/kubevirt/project-infra/commit/8cc3f970369c05d9146b6217250093cb6db03504))
is there a reason for this diff?